ui/src/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,15 @@
+import ls from "localstorage-slim";
+import { useEffect, useState } from "react";
+
+export function useLocalStorage<T>(
+  key: string,
+  defaultValue: T
+): [T, React.Dispatch<React.SetStateAction<T>>] {
+  const [value, setValue] = useState(ls.get<T>(key) ?? defaultValue);
+
+  useEffect(() => {
+    ls.set(key, value);
+  }, [key, value]);
+
+  return [value, setValue];
```

Comment:
Assuming we agree to use the helpers from the bridge, does this hook still make sense?

Reply:
I would combine the two, because the point of the hook is that it sets the state as well without having to additionally handle it in the component.

Reply:
Removed as discussed.